cmd/create/create.go

```diff
@@ -0,0 +1,33 @@
+package create
+
+import (
+	cmdcore "github.com/lyft/flytectl/cmd/core"
+	"github.com/spf13/cobra"
+)
+
+// Long descriptions are whitespace sensitive when generating docs using sphinx.
+const (
+	createCmdShort = `Used for creating various flyte resources including tasks/workflows/launchplans/executions/project.`
+	createCmdLong  = `
+Example Create projects.
+::
+
+bin/flytectl create project --help
+
+`
+)
+
+// CreateCommand will return create command
+func RemoteCreateCommand() *cobra.Command {
```

Comment:
```suggestion
func CreateCommand() *cobra.Command {
```

Reply:
it cause lint error. We cant use package name as prefix